kits/democracy/scripts/democracy-deploy.js

```diff
@@ -0,0 +1,14 @@
+const deployKit = require('@aragon/kits-beta-base/scripts/deploy_kit.js')
+
+module.exports = async (callback) => {
+  const deployConfig = {
```

Comment:
Just curious, where are you defining `ENS` for `deployKit` when you run `deploy:kit:rpc`? `ensAddress` nor `network` are being passed, so I understand it should be in `process.ENV`, but I don't see where it's set.

Reply:
Yes, it assumes `ENS` has been passed as an environment variable. In the case of aragen it gets exported here: https://github.com/aragon/aragen/blob/863cd750d8d5c326c624ebee13362d72b316b406/scripts/set-env